CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Added
+- A `from_xxx` classmethod for every `read_xxx` method, to enable instantiation from
```

Comment:
Can you maybe list the new methods for the reader who is newish to the package?

Reply:
I changed the statement to have all the classmethods I added.